eisbuk-admin/functions/src/migrations.ts

```diff
@@ -383,3 +383,36 @@ export const addIdsToCustomers = functions
       return { success: false };
     }
   });
+
+export const deleteOrphanedBookings = functions
+  .region("europe-west6")
+  .https.onCall(async ({ organization }, { auth }) => {
+    await checkUser(organization, auth);
+
+    const orgRef = admin
+      .firestore()
+      .collection(Collection.Organizations)
+      .doc(organization);
+
+    // get all customers and bookings
+    const allCustomers = await orgRef
+      .collection(OrgSubCollection.Customers)
+      .get();
+    const customerIds = allCustomers.docs.map(({ id }) => id);
+    const bookingsRef = orgRef.collection(OrgSubCollection.Bookings);
+    const allBookings = await bookingsRef.get();
+
+    const toDelete: Promise<any>[] = [];
+    allBookings.forEach((doc) => {
+      const docRef = doc.ref;
```

Comment:
Very nitpicky, but I'll still mention this. No need to change (it can go in as is).

This line is here so that `docRef` can be used as a name, maybe because it's more descriptive, or shorter than `doc.ref`. I believe this line is not needed, and we can use `doc.ref` later in this function.

Reply:
+1